prov/gni/src/gnix_ep.c

```diff
@@ -501,6 +881,7 @@ static int gnix_ep_control(fid_t fid, int command, void *arg)
 		if (ep->type == FI_EP_RDM) {
 			dom = ep->domain;
 			for (i = 0; i < dom->fabric->n_wc_dgrams; i++) {
+				assert(ep->recv_cq != NULL);
```

Comment:
should we check here for a send cq being bound to the ep as well? Doing that here would eliminated the need for subsequent checks when RMA ops, etc. are invoked.